app/models/spotlight/resources/oaipmh_mods_parser.rb

```diff
@@ -189,5 +193,55 @@ def transform_to_iiif_thumbnail(ids_uri)
       uri = uri.gsub(/\/full\/\d*,\d*\/0\/default.jpg/, '')
       uri += "/full/#{thumbnail_size},/0/native.jpg"
     end
+
+    # Spotlight v3.3.0
+    # Spotlight expects "exhibit-specific fields" (a.k.a. Exhibit#custom_fields) to not have
+    # a Solr suffix (e.g. _tesim, _ssim, etc.). This method assumes all non-configured fields
+    # are custom and thus removes their Solr suffix when adding them to the @item_sidecar hash.
+    # Configured fields are added as-is (Solr suffix included).
+    def assign_item_sidecar_data(field_name, value)
+      if configured_field_names.include?(field_name)
+        item_sidecar[field_name] = value
+      else
+        custom_field_slug = field_name.sub(/_.*$/, '')
+        item_sidecar[custom_field_slug] = value
+      end
+    end
+
+    # Spotlight v3.3.0
+    # Used to update an existing sidecar's data when harvesting (see
+    # Spotlight::OaipmhHarvester#harvest_item). Default "configured" fields are expected
+    # to be nested in a "configured_fields" sub-hash. This method assumes non-configured
+    # fields are "exhibit-specific fields" (a.k.a. Exhibit#custom_fields) and puts them
+    # in the "top level" of the hash (where Spotlight expects them to be).
+    #
+    # Example:
+    # {
+    #   'configured_fields' => {
+    #     'full_title_tesim' => 'My Title'
+    #   },
+    #   'custom-field' => 'Hello world'
+    # }
+    #
+    # @return [Hash] Sidecar data organized in the format that Spotlight expects
+    def organize_item_sidecar_data
```

Comment:
Refer to #2 under the "Long version" section of this PR's description for an explanation